node/src/types/chainspec/protocol_config.rs

```diff
@@ -36,12 +42,38 @@ impl Display for ActivationPoint {
     }
 }
 
+/// Type storing the information about modifications to be applied to the global state.
+/// It stores the serialized `StoredValue`s corresponding to keys to be modified.
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize, DataSize, Debug)]
+pub struct GlobalStateUpdate(pub(crate) BTreeMap<Key, Vec<u8>>);
+
+impl ToBytes for GlobalStateUpdate {
```

Comment:
I know it's a trivial impl, but could we have a round trip test using `bytesrepr::test_serialization_roundtrip()` please?

Reply:
That causes some problems because there is currently no way to generate a random `Key`.

Should I implement one? If yes, should it be via `rand` traits, or a `random` method? (I'd think the former, because if I'm not mistaken, `TestRng` is not available in the `casper_types` crate).

Reply:
Yes, it'll need to be via the `rand` traits.